x/ccv/consumer/module.go

```diff
@@ -107,6 +107,9 @@ func (AppModule) RegisterInvariants(ir sdk.InvariantRegistry) {
 // RegisterServices registers module services.
 func (am AppModule) RegisterServices(cfg module.Configurator) {
 	consumertypes.RegisterQueryServer(cfg.QueryServer(), am.keeper)
+	cfg.RegisterMigration(consumertypes.ModuleName,
+		1, // from version 1
+		am.keeper.MigrateConsumerPacketData)
```

Comment:
Use SDK model https://github.com/cosmos/cosmos-sdk/blob/main/x/staking/keeper/migrations.go

Reply:
You meant just pattern matching their top level migration funcs? https://github.com/cosmos/interchain-security/commit/a21eefcbb25285bdadf59952b9637a7c10e79243

Reply:
Yeah, but then you need to register Migrate1to2 here.

Reply:
https://github.com/cosmos/interchain-security/commit/0aa82001c6cf3945f58f34b07500e51e1595a14a